src/main/java/uk/gov/pay/ledger/performance/dao/PerformanceDao.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.pay.ledger.performance.dao;
```

Comment:
I think this could all live under the 'report' package, all of these routes will provide higher level aggregate info and having it all in the same place seems to make sense.

Either this could be part of the `ReportDao` directly or it could be a new Dao still under the report package.